sqlx-core/src/any/arguments.rs

```diff
@@ -1,13 +1,16 @@
 use crate::any::value::AnyValueKind;
-use crate::any::Any;
+use crate::any::{Any, AnyTypeInfoKind};
 use crate::arguments::Arguments;
 use crate::encode::{Encode, IsNull};
 use crate::error::BoxDynError;
 use crate::types::Type;
+use std::collections::HashMap;
 
 pub struct AnyArguments<'q> {
     #[doc(hidden)]
     pub values: AnyArgumentBuffer<'q>,
+    #[doc(hidden)]
+    pub null_values: HashMap<usize, AnyTypeInfoKind>,
```

Comment:
Instead of a separate hashmap, just change `AnyValueKind::Null` to be `Null(AnyTypeInfoKind)`.

Reply:
I was aiming for the least disruptive change but I agree that this is better.